certifi/core.py

```diff
@@ -4,12 +4,26 @@
 certifi.py
 ~~~~~~~~~~
 
-This module returns the installation location of cacert.pem.
+This module returns the installation location of cacert.pem or its contents.
 """
 import os
 
+try:
+    from importlib.resources import read_text
+except ImportError:
+    # This fallback will work for Python versions prior to 3.7 that lack the
+    # importlib.resources module but relies on the existing `where` function
+    # so won't address issues with environments like PyOxidizer that don't set
+    # __file__ on modules.
+    def read_text(_module, _path, encoding="ascii"):
+        return open(where(), "r", encoding=encoding).read()
```

Comment:
Please use a with statement so the file is closed promptly/this doesn't generate a warning.

Reply:
Done.